rkt fly panic

```
$ sudo ./rkt run --interactive --insecure-options=image --stage1-path=./stage1-fly.aci --volume data,kind=host,source=/mnt,readOnly=false --mount volume=data,target=/mnt docker://fedora --exec /bin/bash
image: using image from file ./stage1-fly.aci
image: using image from local store for url docker://fedora
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x404013]

goroutine 1 [running]:
main.evaluateMounts(0xc8200b73b0, 0x26, 0xc8200c2420, 0x6, 0xc82001a540, 0x0, 0x0, 0x0, 0x0, 0x0)
	/opt/build-rkt/gopath/src/github.com/coreos/rkt/stage1_fly/run/main.go:125 +0x1d83
main.stage1(0xc8200c2020)
	/opt/build-rkt/gopath/src/github.com/coreos/rkt/stage1_fly/run/main.go:248 +0xa65
main.main()
	/opt/build-rkt/gopath/src/github.com/coreos/rkt/stage1_fly/run/main.go:367 +0xaa

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/lib/go/src/runtime/asm_amd64.s:1721 +0x1
```

--- a/stage1_fly/run/main.go
+++ b/stage1_fly/run/main.go
@@ -112,6 +112,11 @@ func evaluateMounts(rfs string, app string, p *stage1commontypes.Pod) ([]flyMoun
 	imApp := p.Images[app].App
 	namedVolumeMounts := map[types.ACName]volumeMountTuple{}
 
+	var manifestMPs []types.MountPoint
+	if imApp != nil {
+		manifestMPs = imApp.MountPoints
+	}
+
 	for _, m := range p.Manifest.Apps[0].Mounts {
 		_, exists := namedVolumeMounts[m.Volume]
 		if exists {
@@ -122,7 +127,7 @@ func evaluateMounts(rfs string, app string, p *stage1commontypes.Pod) ([]flyMoun
 	}
 
 	// Merge command-line Mounts with ImageManifest's MountPoints
-	for _, mp := range imApp.MountPoints {
+	for _, mp := range manifestMPs {
 		tuple, exists := namedVolumeMounts[mp.Name]
 		switch {
 		case exists && tuple.M.Path != mp.Path:
@@ -148,7 +153,7 @@ func evaluateMounts(rfs string, app string, p *stage1commontypes.Pod) ([]flyMoun
 	}
 
 	// Merge command-line Volumes with ImageManifest's MountPoints
-	for _, mp := range imApp.MountPoints {
+	for _, mp := range manifestMPs {
 		// Check if we have a volume for this mountpoint
 		tuple, exists := namedVolumeMounts[mp.Name]
 		if !exists || tuple.V.Name == "" {

